src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -46,14 +46,15 @@ public void Dispose()
 
       public virtual void Wait(CancellationToken token)
```

Comment:
yeah, there's not much information on that parameter. All the samples I reviewed show this parameter as false. It's called 'ownsHandle' and I suppose that's because the handle is actually owned by the process.